src/main/java/com/snowflake/kafka/connector/internal/telemetry/TelemetryConstants.java

```diff
@@ -46,4 +47,12 @@ public final class TelemetryConstants {
   public static final String IS_REUSE_PIPE = "is_reuse_pipe";
   public static final String FILE_COUNT_RESTART = "file_count_restart";
   public static final String FILE_COUNT_REPROCESS_PURGE = "file_count_reprocess_purge";
+
+  public class Streaming {
+    public static final String OFFSET_PERSISTED_IN_SNOWFLAKE = "persisted-in-snowflake-offset";
```

Comment:
1. remove inner class. previously recommendation was to have a separate section/heading in this class which distinguishes between snowpipe and streaming. 
2. lets clarify what tp here means. it is not intuitive but hopefully they read code and understand what tp means.

Reply:
1. discussed offline, added as a comment section
2. I'll expand TP -> TopicPartition